proto/abis/massa/abi/v1/abi.proto

```diff
@@ -566,43 +566,43 @@ message AppendDataRequest {
 }
 
 // Append data result
-message AppendDataResult {}
+message AppendDsValueResult {}
 
 // Get data request
-message GetDataRequest {
+message GetDsValueRequest {
   // key
   bytes key = 1;
   // Address to get data for, if none, use current address
   optional google.protobuf.StringValue address = 2;
 }
 
 // Get data result
-message GetDataResult {
+message GetDsValueResult {
   // value
   bytes value = 1;
 }
 
 // Delete data request
-message DeleteDataRequest {
+message DeleteDsEntryRequest {
   // key
   bytes key = 1;
   // Address to delete data for, if none, use current address
   optional google.protobuf.StringValue address = 2;
 }
 
 // Delete data result
-message DeleteDataResult {}
+message DeleteDsEntryResult {}
 
 // Has data request
-message HasDataRequest {
+message DsEntryExistsRequest {
   // key
   bytes key = 1;
   // Address to check data for, if none, use current address
   optional google.protobuf.StringValue address = 2;
 }
 
 // Has data result
-message HasDataResult {
+message DsEntryExistsResult {
```

Comment:
When the naming change is important like here, also update the comment.

Reply:
such a useless comment should be dropped :/